app/Http/Resources/ShowDoctorResource.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class ShowDoctorResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
+            'name' => $this->name,
+            'email' => $this->email,
+            'specialization' => $this->specialization->name,
+            'photo' => $this->photo,
+            'degree_copy' => $this->degree_copy,
```

Comment:
Return if the doctor is activated or not & registration date ( and in unix timestamp)